.gitignore

```diff
@@ -1,4 +1,5 @@
 .editorconfig
+.DS_STORE
```

Comment:
@bdenham this should be added in your global gitignore see https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Ah cool. Hah! I was wondering why it kept getting removed before. Makes sense now. Thanks for letting me know!